templates/etc/default/docker.j2

```diff
@@ -40,6 +40,20 @@
 {%     set _ = docker__tpl_options.append("--label " + key + '="' + value + '"') %}
 {%   endfor %}
 {% endif %}
+{% if docker__registry_mirrors|d() %}
+{%   for registry_mirror in docker__insecure_registries %}
```

Comment:
You use the wrong list variable as the data source, the `for` loop should use the `docker__registry_mirrors` variable instead.